src/components/Navigation.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUser } from '@fortawesome/free-solid-svg-icons';
 
 const links = [
   { path: '/', text: 'HOME' },
```

Comment:
- [ **OPTIONAL ] Good job fixing all the changes mentioned by the previous reviewer. However, I think it would be better if you change the `Home` text to `Books` as the page is for the books list and to match the Zeplin design and for the project requirements